go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -66,6 +66,7 @@ var (
 
 const (
 	streamModeTar        = "tar"
+	writerBufferSize     = 2 * 1024 * 1024 /*2 MiB*/
```

Comment:
why was this moved from builtin to this file? It's not actually used by xtrabackupengine AFAICT.

Reply:
ah it was already being used. And it is no longer used in builtin. 👍